src/ledger/orderbook.ts

```diff
@@ -125,5 +73,22 @@ export async function getOrderbook(
     reverseOfferResults,
     (reverseOfferResult) => reverseOfferResult.result.offers,
   )
-  return formatBidsAndAsks(orderbook, [...directOffers, ...reverseOffers])
+  // Sort the orders
+  // for both buys and sells, lowest quality is closest to mid-market
+  // we sort the orders so that earlier orders are closer to mid-market
+
+  const orders = [...directOffers, ...reverseOffers]
+  // separate out the orders amongst buy and sell
+  const buy: BookOffer[] = []
+  const sell: BookOffer[] = []
+  orders.forEach((order) => {
+    if (order.Flags === orderFlags.Sell) {
```

Comment:
There could be more than one flag on the order - I believe `order.Flags & orderFlags.Sell === 0` does what you want it to do